pvanalytics/quality/outliers.py

```diff
@@ -57,7 +62,25 @@ def zscore(data, zmax=1.5):
         outlier.
 
     """
-    return pd.Series((abs(stats.zscore(data)) > zmax), index=data.index)
+    data = data.copy()
+    nan_mask = pd.Series([False] * len(data))
+
+    if data.isnull().values.any():
+        if nan_policy == 'raise':
+            raise ValueError("The input contains nan values.")
+        elif nan_policy == 'omit':
+            nan_mask = data.isna()
+        else:
+            raise ValueError("Incorrect specification passed " +
+                             "to zscore's nan_policy.")
```

Comment:
Add the valid options to the error message, and remove the name of the function (it will already be visible in the stack trace).

Also, don't need `+` for strings, python will concatenate adjacent strings automatically in this context:
```suggestion
            raise ValueError("Incorrect specification passed "
                             "to zscore's nan_policy.")
```

Reply:
> Add the valid options to the error message, and remove the name of the function.

Can you add this as well? It really makes a difference for users to be able to address errors quickly.

Reply:
Incorporated! Thanks!